package.json

```diff
@@ -72,11 +72,6 @@
     "delay": 2500,
     "ext": "js,json,html,njk"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged && tsc && npm test"
-    }
-  },
   "lint-staged": {
```

Comment:
we can remove this block too since we are no longer using it

we can also remove the dev dependencies on husky i think